UID2.Client.nuspec

```diff
@@ -2,19 +2,21 @@
 <package>
   <metadata>
     <id>UID2.Client</id>
-    <version>2.0.2</version>
-    <title>UID2 C# DSP Client Library</title>
+    <version>4.0.0</version>
+    <title>UID2 Client C# SDK</title>
     <authors>UID2 team</authors>
     <owners>UID2 team</owners>
     <license type="expression">Apache-2.0</license>
     <projectUrl>https://github.com/IABTechLab/uid2-client-net</projectUrl>
     <icon>icon.png</icon>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>Client for working with advertising UID2 services</description>
-    <copyright>Copyright 2021 The Trade Desk, Inc</copyright>
+    <copyright>Copyright 2023 The Trade Desk, Inc</copyright>
     <tags>UID2</tags>
+    <readme>README.md</readme>
+    <repository type="git" url="https://github.com/IABTechLab/uid2-client-net.git" />
     <dependencies>
-      <group targetFramework="netstandard">
+      <group targetFramework="netstandard2.0">
         <dependency id="NETStandard.Library" version="[2.0.0, )" />
```

Comment:
Should we also list dependency on BouncyCastle?

Reply:
https://stackoverflow.com/questions/67110161/nuspec-dependencies-necessary-with-packagereferences
don't think so as uid2.client.csproj's <PackageReference> should sort out the bouncyCastle and NewtonJson/etc dependency